client/pkg/requirements.pip

```diff
@@ -9,7 +9,7 @@ twisted
 
 # leap deps -- bump me!
 leap.common
-leap.soledad.common>=0.6.0
+leap.soledad.common>=0.6.1
```

Comment:
don't update this until next version is released, `VERSION_COMPAT` file should be enough